processing/data_collection/gazette/spiders/base.py

```diff
@@ -24,24 +24,30 @@ class FecamGazetteSpider(scrapy.Spider):
     total_pages = None
 
     def start_requests(self):
-        if self.total_pages is None:
-            yield scrapy.Request(
-                f"{self.URL}?q={self.FECAM_QUERY}", callback=self.parse
+        yield scrapy.Request(
+            f"{self.URL}?q={self.FECAM_QUERY}", callback=self.first_parse
+        )
+
+    def first_parse(self, response):
```

Comment:
Maybe something like `parse_pages` (or `parse_pagination`) and `parse_gazettes` would be a better name? `first_parse` doesn't tell me much.

Reply:
Fixed!